### PR TITLE
Adding SS4

### PR DESCRIPTION
**Pull request by rjavila**
Adding preliminary data and scripts for Science Saturday 4, where we tested chain pizzas.

### CODE REVIEW

File: ScienceSaturday04/pizza_results.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+def read_tables():
+
+    #Reading in cheese results and creating pandas dataframe
+    f1  = pd.read_excel('SS4_pizza_tables.xlsx',
+                        sheet_name='Cheese Results',
+                        header=0,
+                        usecols='C:K'
+                        )
+
+    pizzas = ['Pappa John\'s','Pizza Boli\'s','Digiorno','Little Caesar\'s','Dominos','Pizza Hut','Pizza Hut Stuffed Crust']
```
Comment (dborncamp) [inline ScienceSaturday04/pizza_results.py]: No need for escape characters for apostrophes if you use double quotes (`" ' "`)

File: ScienceSaturday04/pizza_results.py [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+                         sheet_name='Cheese temperatures',
+                         header=1
+                         )
+
+    tab4 = pd.read_excel('SS4_pizza_tables.xlsx',
+                         sheet_name='Pepperoni temperatures',
+                         header=1
+                         )
+    return tab1,tab2,tab3,tab4
+    return tab1,tab2
```
Comment (dborncamp) [inline ScienceSaturday04/pizza_results.py]: Why 2 returns?

**Review by dborncamp — APPROVED** [review]
Not very PEP8 compliant, but seems to work. Interesting results!

**Comment by rjavila** [comment]
PEP-8 compliance? It's a repository about how we drink booze.

File: ScienceSaturday04/pizza_results.py (hunk 1 above)
Comment (rjavila) [reply ScienceSaturday04/pizza_results.py]: Changed those.

File: ScienceSaturday04/pizza_results.py (hunk 2 above)
Comment (rjavila) [reply ScienceSaturday04/pizza_results.py]: Fixed that. Now returns 4 tables.

**Comment by dborncamp** [comment]
Fair point. It's also about eating copious amounts of food...

**Pull request merged**